packages/server/src/index.js

```diff
@@ -113,6 +113,6 @@ exports.start = _app => {
 /**
  * Alias to Bus method.
  */
-exports.registerApi = (id, api) => {
-    bus.registerApi(id, api)
+exports.registerApi = (id, api, mode) => {
```

Comment:
I've no idea why I forgot to provide the same interface :/ to avoid this kind of divergence, we can even do:

```javascript
exports.registerApi = bus.registerApi.bind(bus)
```

What do you think?

Reply:
Perfect, I'm going to update this :)